core/new-gui/src/app/dashboard/user/type/user-project.ts

```diff
@@ -1,9 +1,8 @@
-export interface UserProject
-  extends Readonly<{
-    pid: number;
-    name: string;
-    description: string;
-    ownerID: number;
-    creationTime: number;
-    color: string | null;
-  }> {}
+export interface UserProject {
+  pid: number;
+  name: string;
+  description: string;
+  ownerID: number;
+  creationTime: number;
+  color: string | null;
```

Comment:
Read-only properties are usually used to protect variables from unexpected change that interferes with the inner workings of a class.
This is a Plain Old JavaScript Object (POJO), there is no need to make them read-only.

Reply:
The reason for removing readonly is to update the properties when an update request is successfully processed by the server.

Reply:
Makes sense